public_html/wp-content/plugins/wcpt/wcpt-wordcamp/wordcamp-new-site.php

```diff
@@ -842,7 +842,7 @@ public static function get_stub_me_sponsors_meta( $assigned_sponsor ) {
 			'twitter_handle', 'street_address1', 'street_address2', 'city', 'state', 'zip_code', 'country',
 		);
 
-		switch_to_blog( BLOG_ID_CURRENT_SITE ); // Switch to central.wordcamp.org.
+		switch_to_blog( WORDCAMP_ROOT_BLOG_ID ); // Switch to central.wordcamp.org.
```

Comment:
These are probably not needed, but they do make the code more readable/predictable.